src/components/AbstractPageDefinitions.js

```diff
@@ -436,6 +442,70 @@ export default class AbstractPageDefinitions extends Component {
     )
   }
 
+  showVersionSelectorPopup(component, multiple) {
+    this.setState({ showVersionSelectorPopup: true, multipleVersionSelection: multiple, selectedComponent: component })
+  }
+
+  applySelectedVersions(versions) {
+    const { multipleVersionSelection, selectedComponent } = this.state
+    if (!multipleVersionSelection) {
+      return this.setState({ showVersionSelectorPopup: false }, async () => {
+        if (selectedComponent.changes) {
+          const key = `open${Date.now()}`
+          const NotificationButtons = (
+            <Fragment>
+              <AntdButton
+                type="primary"
+                size="small"
+                onClick={async () => {
+                  await this.onRemoveComponent(selectedComponent)
+                  await this.onAddComponent(
+                    EntitySpec.fromCoordinates({ ...selectedComponent, revision: versions, changes: {} })
+                  )
+                  notification.close(key)
+                }}
+              >
+                Confirm
+              </AntdButton>{' '}
+              <AntdButton type="secondary" size="small" onClick={() => notification.close(key)}>
+                Dismiss
+              </AntdButton>
+            </Fragment>
+          )
+          notification.open({
+            message:
+              'This definition has some changes. Do you confirm to change the version? All the unsaved changes will be lost.',
```

Comment:
Simplify to `This definition has some changes. All the unsaved changes will be lost.`